src/miner.cpp

```diff
@@ -183,6 +186,9 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const CScript& s
 
             const CTransaction& tx = iter->GetTx();
 
+            if (!fIncludeWitness && !tx.wit.IsNull())
```

Comment:
@sipa I'm not sure it's necessary to replace it with anything.  Given the way versionbits soft forks activate, its not really possible to have it unactivate without an enormous reorg